From 6acf166541cbd3d6f6e627f069e3b7130b5bb578 Mon Sep 17 00:00:00 2001
From: Stephan Hartmann <stha09@googlemail.com>
Date: Thu, 6 Oct 2022 10:47:38 +0000
Subject: [PATCH] libstdc++: use math.h for atan2f in gfx::LabToLCH

libstdc++ does not implement std::atan2f. Use atan2f from math.h
instead.
---
 ui/gfx/color_conversions.cc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/ui/gfx/color_conversions.cc b/ui/gfx/color_conversions.cc
index fc2eff3..34a032f 100644
--- a/ui/gfx/color_conversions.cc
+++ b/ui/gfx/color_conversions.cc
@@ -180,7 +180,7 @@ std::tuple<float, float, float> LchToLab(float l,
 }
 std::tuple<float, float, float> LabToLCH(float l, float a, float b) {
   return std::make_tuple(l, std::sqrt(a * a + b * b),
-                         gfx::RadToDeg(std::atan2f(b, a)));
+                         gfx::RadToDeg(atan2f(b, a)));
 }
 
 std::tuple<float, float, float> LabToXYZD50(float l, float a, float b) {
-- 
2.35.1

